src/GrumPHP/Task/Composer.php

```diff
@@ -85,5 +87,28 @@ public function run(ContextInterface $context)
         if (!$process->isSuccessful()) {
             throw new RuntimeException($process->getOutput());
         }
+
+        if ($config['no_local_repository']) {
+            $this->checkLocalRepository($files->first());
+        }
+    }
+
+    protected function checkLocalRepository($composerFile)
```

Comment:
There are some issues with this function:
- no docblocks
- protected instead of private
- no explicit argument types.
- Use sprintf for the RuntimeException.
- Better error message: which local reposities are available in the composer file?